client/my-sites/theme/main.jsx

```diff
@@ -815,4 +816,4 @@ export default connect(
 		setThemePreviewOptions,
 		recordTracksEvent,
 	}
-)( localize( ThemeSheetWithOptions ) );
+)( localize( withPerformanceTrackerStop( ThemeSheetWithOptions ) ) );
```

Comment:
@scinos: Is it safe to apply this here, or would it be better to apply to one of the inner components?

Reply:
Sergio is AFK today so I can chime in. I think because there aren't any delayed renderings further deep down in the component tree, you're safe where you have it becuase when ThemeSheetWithOptions is rendered the page will be interactive. If there were parts of the page that were waiting for data to come down then you'd want to use the component.

Reply:
Thank you for confirming, @saramarcondes! 👍

Reply:
Thanks, @scinos! I thought we'd want to measure both cases, but I suppose only successful theme page renders are really relevant.

Done.